python/equistore-operations/tests/equal_metadata.py

```diff
@@ -197,28 +196,28 @@ def test_single_nonexisting_meta(test_tensor_map_1, test_tensor_map_2):
     # wrong metadata key alone
     wrong_meta = "species"
     error_message = f"Invalid metadata to check: {wrong_meta}"
-    with pytest.raises(ValueError, match=error_message):
-        equistore.equal_metadata(
```

Comment:
I don't know how was this passing the tests before 🤣

Reply:
I think it was raising a ValueError before the PR. In my first commit I changed it to NotEqualError

Reply:
Ahh okay!